UPGRADE-4.2.md

```diff
@@ -261,6 +261,7 @@ TwigBundle
 Validator
 ---------
 
+ * The strict, checkMX, and checkHost properties of the Email validator are deprecated
```

Comment:
`strict` is already mentioned in UPGRADE-4.1 :) also i suggest to use backticks

Reply:
fixed